include/roboteam_ai/treeinterp/BTFactory.h

```diff
@@ -47,10 +49,17 @@ class BTFactory {
 
         static bool hasMadeTrees();
 
-    private:
+
+
+private:
         static std::string currentTree;
         static std::string keeperTree;
         static bool weMadeTrees;
+
+
+
+        static std::shared_ptr<bt::BehaviorTree> testing_tree;
```

Comment:
Also it would be nice if you have a comment which explains the meaning of it. Moreover prevent static variables where possible.